receiver/azuremonitorreceiver/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.17.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.8.2
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/monitor/armmonitor v0.11.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v1.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v2.0.0
```

Comment:
https://github.com/Azure/azure-sdk-for-go/releases/tag/sdk%2Fresourcemanager%2Fresources%2Farmresources%2Fv2.0.0